assets/launcher/lang/sklmessages_es_MX.properties

```diff
@@ -70,36 +70,36 @@ about.translation.authors=SrRapero720
 
 # Login Screen
 login.button.type.microsoft=Acceder con Microsoft
-login.button.type.offline=Acceder Offline
+login.button.type.offline=Iniciar Sesion
 login.label.username=Usuario
-login.dropdown.name=Usuarios con Session activa
+login.dropdown.name=Sesiones activas
 login.button.play=Jugar
 login.button.logout=Cerrar session
-login.button.switch.offline=Cambiar a modo Offline
-login.button.switch.online=Cambiar a modo Online
+login.button.switch.offline=Modo offline (sin cuenta)
+login.button.switch.online=Modo Online (microsoft)
 
 # Ajustes
 settings.header.general=General
-settings.header.updates=Actualizaciones
+settings.header.updates=Registro de cambios
 settings.header.about=Acerca de
 settings.footer.button.save=Guardar
 
-settings.option.language.name=Lenguaje
-settings.option.language.info=Necesitas reiniciar el launcher para que surta efecto
-settings.option.accent.name=Color principal
-settings.option.theme.name=Temas
-settings.option.theme.dark.name=Oscuro
-settings.option.sorting.name=Ordenar Instancias
-settings.option.sorting.bylastplayed.name=Ordenar por ultima vez jugado
-settings.option.sorting.byname.name=Ordenar por Nombre
+settings.option.language.name=Idioma
+settings.option.language.info=Debes reiniciar el launcher para que haga efecto el cambio
+settings.option.accent.name=Color Tematico
+settings.option.theme.name=Tema Oscuro
+settings.option.theme.dark.name=Activar
+settings.option.sorting.name=Orden de las Instancias
+settings.option.sorting.bylastplayed.name=Ultima vez jugado
+settings.option.sorting.byname.name=Nombre
 settings.option.default.name=Instancia por defecto
-settings.option.default.hide.name=Ocultar Instancia por defecto
+settings.option.default.hide.name=Ocultar
 settings.option.console.name=Consola del launcher
 settings.option.console.show.name=Mostrar
-settings.about.contributors=Gracias a todos nuestros contribuyentes:
+settings.about.contributors=Gracias a todos los que nos apoyan:
```

Comment:
I don't think this is correct. Contributor and Supporter mean different things. Contributor is the person who made real changes to the project.

Reply:
No take literal translate
Apoyar its used to refer to do stuff, contribuir (or contribuyentes) is more for "i donate you money"
when i write that i figure in first instance all on that list are patreons or someting else giving money

Reply:
Take in mind spanish is more "volatil"
for example, "COJER" in spain means "TAKE" but in LatinAmerica means "FUCK" (but less agresive)

Reply:
Also, most google translations are interpretated in a cultural space